pages/es/lb4/iniciando.md

```diff
@@ -0,0 +1,111 @@
+---
+lang: es
+title: 'Iniciando'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar_es
+permalink: /doc/es/lb4/iniciando.html
```

Comment:
I remember that you had [this page](https://loopback.io/doc/es/lb4/Getting-started.html) translated last time. 
I think that one is good enough as we keep the file names and the permalink in English. 
This file can be removed. Thanks!